packages/-ember-data/tests/integration/store-test.js

```diff
@@ -913,19 +913,15 @@ module('integration/store - findAll', function(hooks) {
 
     assert.equal(cars.length, 2, 'It returns all cars');
 
-    let carsInStore = store.peekAll('car');
+    await settled();
```

Comment:
I think we want to structure this a bit differently:

```js
carsPromiseArray = store.findAll('car');

assert.equal(carsPromiseArray.length, 2, 'It returns all cars');
let mini = cars.findBy('id', '1');
assert.equal(mini.model, 'Mini', 'Records have not yet been updated');	    

await carsPromiseArray;
```

after this we would peek and assert the length and ensure that the model has been updated